src/gui/search/searchtab.cpp

```diff
@@ -330,6 +343,12 @@ void SearchTab::fillFilterComboBoxes()
     m_ui->filterMode->setCurrentIndex(index == -1 ? 0 : index);
 }
 
+void SearchTab::filterSearchResults(const QString &name)
+{
+    m_proxyModel->setFilterRegExp(QRegExp(name, Qt::CaseInsensitive, QRegExp::WildcardUnix));
```

Comment:
#8722 and #8723 got me thinking why `QRegExp::WildcardUnix` is used in [TransferListWidget](https://github.com/qbittorrent/qBittorrent/blob/38837db8de898c4c6a7185a4e05fafd5a617c5b0/src/gui/transferlistwidget.cpp#L1186) and i blindly copied here. I think i'll change it (here) to the default `QRegExp::RegExp` if no one objects.
Docs: http://doc.qt.io/qt-5/qregexp.html#PatternSyntax-enum

Reply:
Changed.